core/src/avm2/globals/flash/geom/transform.rs

```diff
@@ -249,3 +249,29 @@ pub fn object_to_matrix<'gc>(
 
     Ok(Matrix { a, b, c, d, tx, ty })
 }
+
+pub fn get_pixel_bounds<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let this = this.unwrap();
+    let display_object = get_display_object(this, activation)?;
+    rectangle_to_object(display_object.world_bounds(), activation)
+}
+
+pub fn rectangle_to_object<'gc>(
```

Comment:
Is there a reason for this function being public?

Reply:
Nope, copied from similar function up the file and forgot to remove, thanks